externals/kyuubi-spark-sql-engine/src/main/scala/org/apache/kyuubi/engine/spark/session/SparkSQLSessionManager.scala

```diff
@@ -157,6 +157,10 @@ class SparkSQLSessionManager private (name: String, spark: SparkSession)
   }
 
   override def closeSession(sessionHandle: SessionHandle): Unit = {
+    val closeGracefully = conf.get(ENGINE_SPARK_SESSION_CLOSE_GRACEFULLY)
```

Comment:
is it possible to make this feature work on all engines ? i.e., move code to `SessionManager.closeSession`

Reply:
+1, we also need to do not close engine operation in kyuubi gateway side.

Reply:
done